src/Symfony/Component/HttpKernel/Tests/Fixtures/BundleDependencies/BundleCDependenciesBA.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\Fixtures\BundleDependencies;
+
+use Symfony\Component\HttpKernel\Bundle\BundleDependenciesInterface;
+
+class BundleCDependenciesBA implements BundleDependenciesInterface
+{
+    public function getBundleDependencies()
+    {
+        return array(
+            'Symfony\Component\HttpKernel\Tests\Fixtures\BundleDependencies\BundleBDependenciesA',
```

Comment:
Ok, thats a fair point, as we should decide on the format now and not change it later. I was thinking separate method, maybe even on a separate `BundleOptionalDependenciesInterface` if this would be done in a followup.